automation/system_test/prepare.py

```diff
@@ -399,16 +399,28 @@ def _enrich_env(self):
         self._env_config["V3IO_API"] = f"https://{v3io_api_host}"
         self._env_config["MLRUN_DBPATH"] = f"https://{mlrun_api_url}"
 
-        self._env_config["mlrun_model_monitoring_tsdb_profile"] = {
-            "type": "v3io",
-            "name": "mm-tsdb-profile",
-            "v3io_access_key": None,
-        }
-        self._env_config["mlrun_model_monitoring_stream_profile"] = {
-            "type": "v3io",
-            "name": "mm-stream-profile",
-            "v3io_access_key": self._env_config["V3IO_ACCESS_KEY"],
-        }
+        # Since the prepare script is shared across branches, two MM configs are set.
+        # Remove the deprecated config when we stop testing 1.7.x.
+
+        # MM infra for < 1.8.0
+        self._env_config["MLRUN_MODEL_ENDPOINT_MONITORING__TSDB_CONNECTION"] = "v3io"
+        self._env_config["MLRUN_MODEL_ENDPOINT_MONITORING__STREAM_CONNECTION"] = "v3io"
+
+        # MM infra for >= 1.8.0
+        self._env_config["mlrun_model_monitoring_tsdb_profile"] = json.dumps(
```

Comment:
why not using capitalization if that is how, by convention, we set the env config?

Reply:
Because it's not really an env var, but a datastore profile configuration/object for the system tests.
There is no matching configuration in `mlrun/config.py`, but it might be added in the future.